rabit/rabit/Goal/Models/GoalEntity.swift

```diff
@@ -34,10 +34,10 @@ final class GoalEntity: Object {
 extension Days {
     
     init(_ list: List<Int>) {
-        self.set = Set(list.compactMap { Day(rawValue: $0) })
+        self.selectedValues = Set(list.compactMap { Day(rawValue: $0) })
     }
     
     func toIntArray() -> [Int] {
-        return Array(self.set).map { $0.rawValue }
+        return Array(self.selectedValues).map { $0.rawValue }
```

Comment:
이 부분도 앞서 말씀드린 것처럼 따로 Array로 변환하지 않고 `map(_:)` 을 바로 사용하시면 자동으로 Array를 반환하게 되는것 같습니다!
```suggestion
        return self.selectedValues.map(\.rawValue)
```

Reply:
아 이거 저번에 알려주셨던거군요 ㅋㅋ 깜빡하고 있었네요!!
commit suggestion된거 바로 커밋해보겠습니다~!